public_html/wp-content/themes/pattern-directory/package.json

```diff
@@ -54,7 +54,7 @@
 	"eslintConfig": {
 		"extends": "../../../../.eslintrc.js",
 		"globals": {
-			"wporgBlockPattern": "readonly"
+			"wporgAssetUrl": "readonly"
```

Comment:
`wporgBlockPattern` was copied over from the eslintConfig in the pattern creator, so it can just be replaced here. it's not used in the theme.